README.md

```diff
@@ -262,6 +262,40 @@ In `rv32emu` repository, there are some prebuilt ELF files for testing purpose.
 * `scimark2.elf` : See [tests/scimark2](tests/scimark2) [RV32MF]
 * `stream.elf` : See [tests/stream](tests/stream.c) [RV32MF]
 
+## Benchmarks
+The benchmarks ([Source code](https://github.com/sysprog21/rv32emu/tree/master/tests)) are classified into three categories based on their characteristics:
```

Comment:
Remove the hyperlink to "Source code" because the prior section already mentions.